tests/BrowserKitConfig.php

```diff
@@ -35,4 +35,20 @@ protected function supportsJs()
     {
         return false;
     }
+
+    public function skipMessage($testCase, $test)
+    {
+        if (
+            'Behat\Mink\Tests\Driver\Form\Html5Test' === $testCase
+            && in_array($test, array(
+                'testHtml5FormAction',
+                'testHtml5FormMethod',
+            ))
+            && !method_exists('Symfony\Component\DomCrawler\Tests\FormTest', 'testGetMethodWithOverride')
```

Comment:
Can't we find a method detecting relying on public classes instead ? test cases can be refactored in any patch release as they are purely internal.

Reply:
I can't see any significant API changes for Browserkit 3.3 that we can latch onto

An alternative would be to use `muglug/PackageVersions` (or the upstream `ocramius` version of the PHP versions work out) in dev to explicitly detect the installed versions

Reply:
Or parse `composer info` in the travis bash, and set an env var etc. to be detected in the script

Reply:
Yup, seems much more complex options. The only sensible alternative (to me) would be a `DISABLE_ACTION_ATTRIBUTES_TEST` environment variable that we simply set in Travis for the "old" builds.